kernel/network/network_task.c

```diff
@@ -21,38 +21,45 @@
 #include "network_task.h"
 #include "kernel/network/e1000.h"
 #include "kernel/logger/logger.h"
+#include "kernel/network/pdu_handler.h"
+#include "kernel/network/routing.h"
 
 #include <sys/types.h>
 #include <string.h>
 
-static int queue_start  = 0;
-static int queue_end    = 0;
-static raw_packet_t packet_queue[MAX_PACKET_COUNT] = { 0 };
+static int recv_queue_start  = 0;
+static int recv_queue_end    = 0;
+static raw_packet_t recv_packet_queue[MAX_PACKET_COUNT] = { 0 };
+
+static int send_queue_start  = 0;
```

Comment:
We could remove these because we are not using them and currently do not plan to do so

Reply:
Yes